classes/person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = nil
```

Comment:
- [ ] Kindly make sure the id is a number generated randomly, you can use `Random.ran` to generate a random id for the person